src/test/java/uk/gov/justice/probation/courtcasematcher/pact/HearingServiceConsumerPactTest.java

```diff
@@ -23,7 +23,7 @@
 @ExtendWith(PactConsumerTestExt.class)
 @PactTestFor(providerName = "court-case-service")
 @PactDirectory(value = "build/pacts")
-class CourtCaseServiceConsumerPactTest {
+class HearingServiceConsumerPactTest {
```

Comment:
CourtCaseService refers to the name of the service so should remain as CourtCaseService